explainers/subresource-loading.md

```diff
@@ -206,10 +206,18 @@ of resources could look like:
 <link
   rel="webbundle"
   href="https://example.com/dir/subresources.wbn"
-  resources="https://example.com/this,https://example.com/is,https://example.com/not,https://example.com/a,https://example.com/real,https://example.com/list,https://example.com/of,https://example.com/URLs"
+  resources="https://example.com/this https://example.com/is https://example.com/not https://example.com/a https://example.com/real https://example.com/list https://example.com/of https://example.com/URLs"
 />
 ```
 
+IDL:
+
+``` webidl
+partial interface HTMLLinkElement {
+  [SameObject, PutForwards=value] readonly attribute DOMTokenList resources;
```

Comment:
What's `value`?

Reply:
Thanks. As @jyasskin suggested, it looks we are not supposed to put WebIDL into explainers at all. So let me remove the IDL section.